shared/src/Datadog.Trace.ClrProfiler.Native/util.h

```diff
@@ -26,13 +29,26 @@ static fs::path GetCurrentModuleFolderPath()
     }
 #else
     Dl_info info;
-    if (dladdr((void*)GetCurrentModuleFolderPath, &info))
+    if (dladdr((void*) GetCurrentModuleFolderPath, &info))
     {
         return fs::path(info.dli_fname).remove_filename();
     }
 #endif
     return {};
 }
+#ifdef _WIN32
+static fs::path GetPoliciesPath()
```

Comment:
Just a thought - I'd suggest that we only keep this calculation as a "fallback" path, and instead that we set this path via an env variable in fleet installer? That way if, for whatever reason, we need to move where the agent puts this, we're still backwards compatible? WDYT @bmermet ?

Reply:
My thinking was that this would be the fleet installer (not the .NET `FleetInstaller.exe` 😅 ) that sets the path. Because the location of the flatbuffers file is essentially managed completely independently, by the agent(?), so what happens if we need to put it somewhere else, or rename it or etc? A new version of the agent/installer might need to change the paths, say for security reasons? 

As an aside, purely for integration testing, it makes this actually testable in CI 😄 So I'd advocate for an env var override either way tbh, even if we want to have the fallback be the "normal" case